05A-foundational-skills-DRAFT.Rmd

```diff
@@ -41,24 +41,471 @@ Make an illustration that shows how these are related, highlighting where in the
     + How do I explore them? (CRAN, vignettes, Help docs, blog posts)
     + When should I write my own?
 
-* Data (nest in DS workflow) - embed chaining functions
-    + How do I import it?
-    + How do I explore it? 
-    + How do I tidy it? 
-    + How do I transform it?
-    + How do I visualize it?
-    + How do I model it?
-    + How do I communicate it?
-    + Do data structures matter?
-    + Do data types matter?
-    + Tidy data
-
 * Functions
     + What are they?
     + Why do I need them?
     + How do I know which arguments to provide?
     + How are they related to packages?
     + When should I write my own?
     + How do I write my own?
+
+* Data 
+    + How does data integrate with packages and functions within a Project?
+
+## Chapter overview
+This chapter is designed to give you the skills and knowledge necessary to _get started_ in any of the walk through chapters. Our goal is to get you working with R using the RStudio *I*ntegrated *D*evelopment *E*nvironment (IDE) through a series of applied examples.  
```

Comment:
I love this because it's a clear statement of this chapter's goal. So as we keep developing it we can go back to this goal as the design principle.